lib/packwerk/cli.rb

```diff
@@ -195,11 +196,20 @@ def parse_run(params)
         ignore_nested_packages = false
       end
 
+      if params.any? { |p| p.include?("--offenses_formatter") }
```

Comment:
```suggestion
      if params.any? { |p| p.include?("--offenses-formatter") }
```
Flags should alway be dasherized (or support both dashes and underscores)